lua/refactoring/treesitter/treesitter.lua

```diff
@@ -196,6 +211,7 @@ local function containing_node_by_type(node, container_map)
     return node
 end
 
+-- TODO: Can we validate settings here without breaking things?
```

Comment:
Does this one break when validated? If so that's weird, we can maybe open an issue for it and tackle it in another PR.

Reply:
Yeah a bunch of tests failed surprisingly, I can open issue for it before merging.